data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,1183 @@
+	npc
+		government "Pirate"
+		personality staying nemesis plunders harvests
+		system "Ultima Thule"
+		fleet "Large Northern Pirates" 1
+		fleet "Small Northern Pirates" 4
+	npc
+		government "Bounty Hunter"
+		personality nemesis unconstrained
+		system Almaaz
+		fleet
+			names "bounty hunter"
+			variant
+				"Leviathan (Particle Chaser)"
+				"Corvette (Particle Chaser)" 2
+
+
+mission "Hai Slave Rescue [01]"
+	name "Hai Slave Rescue: <planet>"
+	description "Take Teeneep to <destination> to seek the missing Hai. Payment is <payment>."
+	landing
+	clearance `You send the clearance code and the pirates give you landing coordinates.`
+	source Freedom
+	passengers 1
+	destination Stormhold
+	blocked `You cannot save the Hai Slaves unless you have one bunk for Teeneep.`
+	to offer
+		has "Hai Slave Rescue [00]: done"
+	on visit
+		dialog `Oopes! You left Teeneep in one of your escorts. Depart and wait for the escorts to arrive.`
+	on offer
+		log `The Hai slaves are on Stormhold, and will not be sold for several months. They're being held for study before sale.`
+		conversation
+			`You reach a building labeled, "Hathor's Exotic Slaves." You were expecting a woman, but Hathor turns out to be a large, fat, hairy, man. He bears no resemblance to his namesake, a famously-beautiful Egyptian goddess.`
+			# Teeneep's intelligence gathering isn't spectacular if she
+			# got Hathor's gender so wildly wrong.
+			
+			`	"You must be Teeneep. Are you here to sell this?" Apparently he's referring to you. "I don't usually deal in dirt belt urchins, but I'll buy this one for 80,000 credits."`
+			# Captain <last> is from a dirt belt planet and worked in a mill.
+			# Not a very exotic slave.
+			
+			`	Teeneep responds, "Actually, I'm looking to buy, not sell. Where can I get Hai slaves? I'll need a lot of them."`
+			# The above line is when we first see Teeneep act completely out-of-character
+			# to fool others into doing what she wants.
+			
+			`	"I've never been fool enough to enslave a Hai. I know some less cautious dealers, and I can inquire for you."`
+			# Apparently slavers view enslaving Hai as being a very bad idea.
+			# They are, perhaps, even more afraid of the Hai than of the Navy.
+			
+			`	Teeneep nods.`
+			`	Hathor vanishes into a nearby office, activates a holo-communicator, and shuts the door.`
+			choice
+				`	(Spy on Hathor's conversation.)`
+			`	It is difficult to make out Hathor's words through the thick door, but you can gather the gist of his conversations. He talks to a few slavers throughout Human space and finds a group near Syndicate space that have hundreds of Hai. They agree to meet you, and Hathor asks for 100,000 credits payment if the two of you are taken as slaves.`
+			choice
+				`	(Continue.)`
+			`	Hathor emerges from his office. "I've scoured the galaxy, and only one slaver has Hai, but they have plenty. I'll want 50,000 credits before I give you details and a clearance code."`
```

Comment:
Didn't I just hear where these slavers were by spying on Hathor? We at least know that they're near Syndicate space. That narrows down our search to 4 planets. We could decline to pay and search on our own.